_extensions/interlinks/interlinks.lua

```diff
@@ -31,7 +31,9 @@ function lookup(search_object)
             if search_object.domain and item.domain ~= search_object.domain then
                 goto continue
             else
-                table.insert(results, item)
+                if item.domain == "py" then
```

Comment:
Could we have this also check `search_object.domain or item.domain == "py"`? That way, people will be able to use the full link syntax still to link to non-python docs (as in [these examples](https://machow.github.io/quartodoc/get-started/interlinks.html#example-python.org-print))

Reply:
Good point, fixed it.